### PR TITLE
Add Next steps to Verify section

### PR DESCRIPTION
**Pull request by beckykd**
Part of https://github.com/Qiskit/documentation/issues/464

### CODE REVIEW

File: docs/verify/simulate-with-qiskit-aer.ipynb
Comment (Eric-Arellano) [inline docs/verify/simulate-with-qiskit-aer.ipynb:1]: Technically this link is to the overall Aer docs, not the API docs. Which did you intend?

File: docs/verify/stabilizer-circuit-simulation.ipynb
Comment (Eric-Arellano) [inline docs/verify/stabilizer-circuit-simulation.ipynb:1]: Ditto that this links to Aer docs, not the specific API docs

**Review by Eric-Arellano — APPROVED** [review]
Thanks!

**Review by abbycross — APPROVED** [review]
👍

**Pull request merged**